docs/RULES.md

```diff
@@ -719,6 +719,8 @@ Aliases: ol-prefix
 
 Parameters: style (`:one`, `:ordered`; default `:one`)
 
+Usage: `rule 'MD029', :style => 'ordered'`
```

Comment:
I'm all for more explicit docs, but this isn't specific to MD029 at all. Look at MD025 above, it's parameter is called `level` and so it's `:level => 2`.

Passing parameters are documented at https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md#parameters 

If that's not clear, I'm certainly open to any changes. But having this under MD029 is more confusing, IMO as it implies MD029 is somehow different from all the others.

Reply:
@jaymzh got you, yeah, it makes sense. I have just walked thru `Creating styles` but I didn't notice paragraph `Parameters` when I was trying to figure out how to manage it to work